debezium-core/src/main/java/io/debezium/pipeline/ChangeEventSourceCoordinator.java

```diff
@@ -435,13 +437,20 @@ public void resumeStreaming() {
 
         @Override
         public void waitSnapshotCompletion() throws InterruptedException {
+            waitSnapshotCompletion(null);
```

Comment:
Just one last nitpick. Can you just use and empty function `() -> {}`?

Reply:
Done